### ISSUE TITLE
Set an output directory for sourcemaps.

### ISSUE THREAD

**Issue by nicolo-ribaudo** [jasonsanjose/brackets-sass#92]
(Maybe duplicate of #61?)

The location of source maps is bound by the location of css files.
If I want to save source maps and css output in two different folders, I have to do something like:
```json
{
    "path": {
        "sass/*.scss": {
            "sass.options": {
                "outputDir": "../css/",
            }
        },
        "sass/file-1.scss": {
            "sourceMap": "../maps/file-1.css.map"
        },
        "sass/file-2.scss": {
            "sourceMap": "../maps/file-2.css.map"
        },
        "sass/file-3.scss": {
            "sourceMap": "../maps/file-3.css.map"
        }
    }
}
```

(The project tree is
```
|
|-- .brackets.json
|-- sass
  |-- file-1.scss
  |-- file-2.scss
  |-- file-3.scss
|-- css
  |-- file-1.css
  |-- file-2.css
  |-- file-3.css
|-- maps
  |-- file-1.css.map
  |-- file-2.css.map
  |-- file-3.css.map
```
)

Would be nice to set the folder only once:
```json
{
    "path": {
        "sass/*.scss": {
            "sass.options": {
                "outputDIr": "../css/",
                "sourceMapDir": "../maps/"
            }
        }
    }
}
```

**Comment by jasonsanjose** [comment]
In PR #100 the `sourceMap` pref can be either `true` or a relative path. Try it out and let me know if it works for you.

**Comment by jasonsanjose** [comment]
So...I realize my previous suggestion isn't quite the same as supporting a `sourceMapDir`. Removing from v2 milestone.
